pkg/services/ngalert/image/service.go

```diff
@@ -122,13 +122,11 @@ func (s *ScreenshotImageService) NewImage(ctx context.Context, r *models.AlertRu
 
 	dashboardUID := r.GetDashboardUID()
 	if dashboardUID == "" {
-		logger.Debug("Cannot take screenshot for alert rule as it is not associated with a dashboard")
```

Comment:
Just to add some background here we added these debug lines because users complained there were no logs when DashboardUID and PanelID were missing. I see the log line deleted, but where is it rephrased? Also, if changing these log lines make sure to update the docs because these log lines are mentioned in the Troubleshooting instructions for Images in notifications.

Reply:
It's not rephrased. I was asked to instead drop it entirely from an [internal discussion](https://raintank-corp.slack.com/archives/C028MCV4R7C/p1678402620314469?thread_ts=1678396478.503169&cid=C028MCV4R7C) cc @yuri-tceretian.

I have zero strong opinions on this one. If this is a blocker for acceptance of this PR i will simply remove the changes in this file.

Reply:
Let's keep them for the time being if possible. I've opened a separate issue here where we can talk about showing this information in Grafana instead https://github.com/grafana/grafana/issues/64633.

Reply:
I think the log message here is too noisy as it is emitted per result (or dimension). Therefore, if you have a rule without dashboard UID annotation that returns 600 results (dimensions) then you will get 600 notifications like this. I think this is counter-productive. 
Also, I think we should not leak knowledge about the rule's domain into the image service domain. There might be some intermediate structure between those two domains. This will make the boundaries much clearer, and naturally move this log message to be somewhere in state manager where it can be emitted only once per rule evaluation.